TRRandomizerView/Controls/EditorControl.xaml.cs

```diff
@@ -158,6 +160,23 @@ public bool Randomize()
             return false;
         }
 
+        try
+        {
+            TRRandomizerCoord.CheckBackupIntegrity();
+        }
+        catch (ChecksumMismatchException)
+        {
+            MessageWindow.ShowError("Game backup data integrity check failed. Randomization cannot be performed as the game data files in the backup directory are not original." +
+                "\n\nPlease uninstall the game and remove any external mods you may have applied.\n\nOnce complete, reinstall the game afresh, and open the data folder again in the randomizer to proceed.",
+                "https://github.com/LostArtefacts/TR-Rando/blob/master/USING.md#troubleshooting");
+
+            if (DeleteBackupImpl())
+            {
+                Unload();
+            }
+            return false;
```

Comment:
Wait, if the game detects a malformed version of the game, it deletes files? IMO it should abort and exit immediately.

Reply:
It's only the backed up files that the rando creates that are deleted, not the game itself. Getting to this point should not be possible any longer because of the hard stop on the first open. The only way would be if you were to open the rando, then manually fiddle with the backup before randomizing.
The problem is that if we don't delete the backup, and the user reinstalls the game, when they open it in the rando again it won't create a new backup because it already exists. And unfortunately we can't rely on the player following an instruction to manually delete the backup.
I agree though, we would never delete the actual target folder as that's well beyond scope, and that is the step users have to perform themselves.